nisystemlink/clients/feeds/models/_feed.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+from ._platform import Platform
+
+
+class Feed(JsonModel):
+    """Feed model."""
+
+    id: Optional[str] = None
+    """The auto-generated ID of the feed."""
+    name: Optional[str] = None
+    """The name of the feed."""
+    description: Optional[str] = None
+    """The description of the feed."""
+    platform: Platform
+    """The platform of the feed, the following package extensions are available: .nipkg for
+    windows feeds, .ipk and .deb for ni-linux-rt feeds.
+    """
+    workspace: Optional[str] = None
+    """The ID of the workspace this feed belongs to."""
+    updated_at: str
```

Comment:
`updated_at` and `created_at` must be `datetime` not `str`.

![image](https://github.com/user-attachments/assets/cb676f01-504e-45bb-a3b4-743fa6c19310)



Please refer to https://github.com/ni/nisystemlink-clients-python/blob/e1dc1c63eca1a60c00d51d4a19ca9daa86f1c7ec/nisystemlink/clients/file/models/_file_metadata.py#L20